src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -776,66 +776,5 @@ internal BoundStatement WrapWithVariablesAndLocalFunctionsIfAny(CSharpSyntaxNode
                                   ImmutableArray.Create(statement))
             { WasCompilerGenerated = true };
         }
-
-        internal string Dump()
```

Comment:
Cyrus

Thanks for the feedback, the tool currently ignores build flavours and thus defines. Based on your comments I need to modify it so that it uses a tool specific build flavour in which all the relevant defines are specified.

Cheers

Zeb